docs/integrations/mirroring.md

```diff
@@ -24,34 +24,34 @@ A Mirroring Integration's YAML file should have the following parameters (under
   isremotesyncout: true
 ```
 Where:
-- isfetch determines if the integration fetches incidents.
-- ismappable determines if the remote schema can be retrieved for this integration.
-- isremotesyncin determines if mirroring from the 3rd party integration to XSOAR is supported.
-- isremotesyncout determines if mirroring from XSOAR to the 3rd party integration is supported.
+- **isfetch** determines if the integration fetches incidents.
+- **ismappable** determines if the remote schema can be retrieved for this integration.
+- **isremotesyncin** determines if mirroring from the 3rd party integration to XSOAR is supported.
+- **isremotesyncout** determines if mirroring from XSOAR to the 3rd party integration is supported.
 
 ## Commands
 Use the following commands to implement a mirroring integration.  
 *Note that when mirroring both incoming and outgoing data, all the commands are required. For mirroring in only one direction, only some of the commands are required.*
 - `test-module` - this is the command that is run when the `Test` button in the configuration panel of an integration is clicked.
 - `fetch-incidents` - this is the command that fetches new incidents to Cortex XSOAR.
-- `get-remote-data` - this command gets new information about the incidents in the remote system and updates *existing* incidents in Cortex XSOAR. If an API rate limit error occures, this method should return the exact error - "API rate limit", so that the sync loop will start from the failed incident.
-This command is executed every 1 minute for each individual **incident** fetched by the integration. 
 - `get-modified-remote-data` - available from Cortex XSOAR version 6.1.0. This command queries for incidents that were modified since the last update. If the command is implemented in the integration, the **get-remote-data** command will only be performed on incidents returned from this command, rather than on all existing incidents. This command is executed every 1 minute for each individual **integration's instance**. 
+- `get-remote-data` - this command gets new information about the incidents in the remote system and updates *existing* incidents in Cortex XSOAR. If an API rate limit error occures, this method should return the exact error - `"API rate limit"`, so that the sync loop will start from the failed incident.
+This command is executed every 1 minute for each individual **incident** fetched by the integration. 
```

Comment:
Probably worth mentioning `sync.mirror.job.enable`, `sync.mirror.job.delay` and `sync.mirror.job.delayAdvanced` somewhere above regarding the job. It's not hard-coded to 1 min.

Reply:
can you explain each of these options here, and I can edit that into the docs. Because I don't familiar with each of these configurations.

Reply:
- `sync.mirror.job.enable` is to enable / disable the mirroring job
- `sync.mirror.job.delay` is the interval for the job in minutes
- `sync.mirror.job.delayAdvanced` is mostly used for demos and overrides `sync.mirror.job.delay` and if specified is the interval for the job in seconds. Idea being that for demos you do not want to wait a min to show mirroring happened.